packages/wxt/src/core/keyboard-shortcuts.ts

```diff
@@ -0,0 +1,50 @@
+import readline from 'node:readline';
+import { WxtDevServer } from '../types';
+
+export interface KeyboardShortcutWatcher {
+  start(): void;
+  stop(): void;
+}
+
+/**
+ * Function that creates a keyboard shortcut handler for the extension.
+ */
+export function createKeyboardShortcuts(
+  server: WxtDevServer,
+): KeyboardShortcutWatcher {
+  let isWatching = false;
+  let rl: readline.Interface | undefined;
+
+  const handleInput = (line: string) => {
+    // Only handle our specific command
+    if (line.trim() === 'o') {
+      server.restartBrowser();
+    }
+  };
+
+  return {
+    start() {
+      if (isWatching) return;
+
+      rl = readline.createInterface({
+        input: process.stdin,
+        output: process.stdout,
+        terminal: false, // Don't intercept ctrl+C, ctrl+Z, etc
```

Comment:
Specifically, I had to add this line, `terminal: false`, for it to work on Mac. Without it, the dev server only stopped after pressing ctrl+C twice. With it, I only have to press ctrl+C once, like before.